hedera-node/hedera-app/build.gradle.kts

```diff
@@ -63,3 +63,91 @@ dependencies {
 tasks.withType<Test> {
     testLogging.exceptionFormat = org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
 }
+
+// Add all the libs dependencies into the jar manifest!
+tasks.jar {
+    doFirst {
+        tasks.jar.configure {
+            manifest {
+                attributes(
+                    "Main-Class" to "com.hedera.node.app.ServicesMain",
+                    "Class-Path" to configurations.getByName("runtimeClasspath")
+                        .joinToString(separator = " ") { "../../data/lib/" + it.name }
+
+                )
+            }
+        }
+    }
+}
+
+// Replace variables in semantic-version.properties with build variables
+tasks.processResources {
+    filesMatching("semantic-version.properties") {
+        filter { line: String ->
+            if (line.contains("hapi-proto.version")) {
+                "hapi.proto.version=" + libs.versions.hapi.version.get()
+            } else if (line.contains("project.version")) {
+                "hedera.services.version=" + project.version
+            } else {
+                line
+            }
+        }
+    }
+}
+
+// Copy dependencies into `data/lib`
+val copyLib = tasks.register<Copy>("copyLib") {
+    from(project.configurations.getByName("runtimeClasspath"))
+    into(project(":hedera-node").file("data/lib"))
+}
+
+// Copy built jar into `data/apps` and rename HederaNode.jar
+val copyApp = tasks.register<Copy>("copyApp") {
+    from(tasks.jar)
+    into(project(":hedera-node").file("data/apps"))
+    rename { "HederaNode.jar" }
+    shouldRunAfter(tasks.getByName("copyLib"))
+}
+
+tasks.assemble {
+    dependsOn(copyLib)
+    dependsOn(copyApp)
+}
+
+// Create the "run" task for running a Hedera consensus node
+tasks.register<JavaExec>("run") {
```

Comment:
I like the idea of creating a "Starter". I just ask if we can externalize this in to a file in the global buildSrc folder and use the "build" directory as a base

Reply:
I agree this would be nice. I would propose to do so in a follow on PR (which you could even take lead on). In this PR, I'm literally just moving what we have in one build.gradle into another, so as to minimize the changes and get the PR merged faster to enable parallel development